firmware/controllers/trigger/trigger_central.cpp

```diff
@@ -75,8 +75,6 @@ void addTriggerEventListener(ShaftPositionListener listener, const char *name, E
 	engine->triggerCentral.addEventListener(listener, name, engine);
 }
 
-uint32_t triggerHanlderEntryTime;
-
```

Comment:
Variable is used only locally, so move it inside (better for register optimization)